package.json

```diff
@@ -6,7 +6,7 @@
   },
   "engines": {
     "npm": ">=6",
-    "node": "12"
+    "node": "15"
```

Comment:
Is this version of node necessary? It doesn't match the one used in the github actions (12.8) can we sync that?

Reply:
No this should not be there, nice catch! thanks!